@here/harp-geoutils/lib/projection/EarthConstants.ts

```diff
@@ -10,4 +10,16 @@ export class EarthConstants {
 
     /** The equatorial radius in meters. */
     static EQUATORIAL_RADIUS: number = 6378137.0;
+
+    /**
+     * Lowest point on dry land: Shore of the Dead Sea.
```

Comment:
While these constants are interesting i'm wondering why we even define them here ?

While max is rather stable, `MIN_ELEVATION` doesn't matter if we choose to visualize depth of oceans ...

Reply:
If we would render ocean elevation, I would assume we would just render shading but not real geometry(except if somebody implements a submarine simulator).
Nevertheless I agree that this is more a data-source specific thing. I will move it.

Reply:
Maybe use them in implementations of `Projections.worldExtent` ?

Reply:
As far as I can see worldExtent is only used for the projection in the tiling scheme, not for the target projection used by MapView(except for the FlatBoundingBoxGenerator that should stay flat).